scalding-date/src/main/scala/com/twitter/scalding/RichDate.scala

```diff
@@ -61,6 +61,10 @@ object RichDate {
   }
 
   def now: RichDate = RichDate(System.currentTimeMillis())
+
+  implicit def richDateOrdering: Ordering[RichDate] = new Ordering[RichDate] {
+    def compare(a: RichDate, b: RichDate) = java.lang.Long.compare(a.timestamp, b.timestamp)
```

Comment:
Should we call the compare function inside the richdate's instead? just so as we avoid any potential future drift.

(Curious I didn't think defining both Ordered and Ordering would get around the bug totally, but good!)